1/contrib/s2i/assemble

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+JENKINS_DIR=/opt/openshift
+JENKINS_INSTALL_DIR=$(mktemp -d --suffix=jenkins)
+
+echo "---> Copying repository files ..."
+cp -Rf /tmp/src/. ${JENKINS_INSTALL_DIR}
+
+# Install jenkins plugins using plugin installer if user has "plugins.txt" file
+# in repository
+if [ -f ${JENKINS_INSTALL_DIR}/plugins.txt ]; then
+  echo "---> Installing Jenkins $(cat ${JENKINS_INSTALL_DIR}/plugins.txt|wc -l) plugins using ${JENKINS_DIR}/plugins.txt ..."
+  /usr/local/bin/plugins.sh ${JENKINS_INSTALL_DIR}/plugins.txt
+  chmod -R g+rw ${JENKINS_DIR}/plugins
```

Comment:
still no fix-permissions script?  i think the directories need to be marked executable if only because @csrwng found that necessary for the rsync logic to work.

though i guess fixing these permissions doesn't matter that much since the files end up copied into the persistent volume...but stil, presumably it would carry the permissions over.